ts/models/messageType.ts

```diff
@@ -118,6 +118,11 @@ export interface DataExtractionNotificationMsg {
   referencedAttachmentTimestamp: number; // the attachment timestamp he screenshot
 }
 
+export enum MessageDirection {
+  outgoing = 'outgoing',
+  incoming = 'incoming',
+}
+
```

Comment:
in typescript, you can just have export type `MessageDirection = 'outgoing' | 'incoming'`
also, this is actually the same as  `MessageModelType`